jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -188,30 +188,41 @@ image::{{site.baseurl}}/assets/img/docs/vscode-ext-config-test-run-crop.png[Scre
 * Your project must **not** make use of xref:dynamic-config#[dynamic configuration]. This feature is disabled for projects that use dynamic configuration.
 
 NOTE: **Unversioned config** in CircleCI indicates that the `.circleci/config.yml` on the branch where the pipeline is running is ignored, and instead, the configuration file is overridden by a custom configuration file passed as a parameter.
 
-[#steps]
-=== Steps
 
 [WARNING]
 ====
-Triggering a pipeline with an _unversioned_ configuration can have side effects, for example, unwanted deployment. We recommend using this feature **only** on feature branches.
+Please be mindful when running pipelines with _unversioned_ configuration, and take care to ensure no jobs with unintended effects are included in the run.
+
+Test run pipelines triggered from VS Code have access to contexts and project environment variables, based on which branch they are run on and on author permissions. We therefore recommend using this feature **only** on feature branches.
+
+This feature is enabled by default on all projects. You can opt-out of this feature per project. For more details, see <<feature-controls>>. 
 
-Test run pipelines triggered from VS Code have access to contexts, and project environment variables, based on which branch they are run on, and user permissions.
+The following issues will be addressed in the coming weeks: 
+- Audit logs do not mention that a pipeline was run with unversioned config.
+- Pipelines run with unversioned configuration are not differentiated from other pipelines in the WebUI.
+- PR status checks may be altered as a result of unversioned config overrides
 ====
 
+[#steps]
+=== Steps
+
 The following steps show how to test run a pipeline from the VS Code extension:
 
 . Open VS Code and select the CircleCI logo in the sidebar. The **Pipelines** and **Run** panels will be visible.
 . Expand the **Run** panel to view your projects.
 . For each project, there is a button to trigger the config test run, and check boxes for each workflow and job for you to choose which parts of your config you want to run. Select your options, and click icon:play[]. Any local change you make to your `.circleci/config.yml` file will be included in the test run.
 . The pipeline is now visible in your link:https://app.circleci.com/pipelines/[dashboard] in the CircleCI web app, and the Pipelines panel in the VS Code extension.
 
+[#feature-controls]
+=== Feature Controls
 
+This feature is enabled by default on all projects, but access to it can be controlled **at the project level**.
 
+* Project level settings can be found under menu:Project Settings[VS Code integration]. Toggle the "Run pipelines with unversioned config" option to **Off** to opt-out. 
 
+We plan on adding organization-level controls for this feature soon. 
```

Comment:
This would be better in the banner of things that are being worked on further up the page. We avoid time sensitive information outside banners to avoid outdated info remaining in the main text

Reply:
How can we achieve this banner? 
To be clear, we will DEFINITELY remove this because it's very much top of mind at the moment.

Reply:
OK if it's very short term then that's fine

Reply:
Turned it into an info banner